tools/arxiv/configs/acid_pipeline_phased.yaml

```
@@ -82,7 +82,10 @@ processing:
   
   # Specific list source
   specific_list:
-    arxiv_ids: []  # Fill in as needed
+    arxiv_ids:
+      - '1301.3781'  # word2vec
+      - '1405.4053'  # doc2vec  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)